includes/class-wc-skroutz-analytics-tracking.php

```diff
@@ -248,4 +253,19 @@ private function calculate_tax_from_total($total, $tax_rate) {
 		return round( $total - $total / ( 1 + $tax_rate/100 ), 2 );
 	}
 
+	/**
+	 * Checks if order is prior to 30 days ago
+	 *
+	 * @return boolean
+	 *
+	 * @since    1.7.0
+	 * @access   private
+	 */
+	private function order_is_obsolete() {
+		if (is_null($this->order->get_date_created())) {
```

Comment:
`get_date_created` seems that is not available in our minimum supported WooCommerce version. 💥 

```
<b>Fatal error</b>
:  Call to undefined method WC_Order::get_date_created() in <b>/var/www/minimum/wp-content/plugins/skroutz-analytics-woocommerce/includes/class-wc-skroutz-analytics-tracking.php</b>
on line <b>265</b>
```

We have to find an alternative

Reply:
The only alternative I found is to use the WC_Order property `order_date` that is actually the `post_date`. It seems that it does not change when the order status is updated, as it updates the property `completed_date`. However this returns a String and not a DateTime object so we have to parse it first.